src/test/java/br/com/six2six/fixturefactory/model/Order.java

```diff
@@ -43,19 +41,19 @@ public void setPayment(Payment payment) {
 		this.payment = payment;
 	}
 
-	public LocalDateTime getRegisterDate() {
+	public java.time.LocalDateTime getRegisterDate() {
 		return registerDate;
 	}
 
-	public void setRegisterDate(LocalDateTime registerDate) {
+	public void setRegisterDate(java.time.LocalDateTime registerDate) {
 		this.registerDate = registerDate;
 	}
 
-	public LocalDate getSendDate() {
+	public java.time.LocalDate getSendDate() {
 		return sendDate;
 	}
 
-	public void setSendDate(LocalDate sendDate) {
+	public void setSendDate(java.time.LocalDate sendDate) {
 		this.sendDate = sendDate;
 	}
```

Comment:
Why did you stop to use simple imports?

Thank you!

Reply:
fixed